src/react/components/common/keyboardManager/keyboardRegistrationManager.ts

```diff
@@ -1,10 +1,13 @@
 import Guard from "../../../../common/guard";
+import { KeyEventType } from "./keyboardManager";
 
 /**
  * A map of keyboard event registrations
  */
 export interface IKeyboardRegistrations {
-    [key: string]: KeyboardEventHandler[];
+    [keyEventType: string]: {
```

Comment:
Should this be a `KeyEventType` ?

Reply:
I wanted to, but it forces me to use number or string